sticker-graph/sticker_graph/rnn.py

```diff
@@ -3,57 +3,54 @@
 import sticker_graph.vendored
 
 
-def dropout_wrapper(
-        cell,
-        is_training,
-        output_keep_prob=1.0,
-        state_keep_prob=1.0):
-    output_keep_prob = tf.cond(
-        pred=is_training,
-        true_fn=lambda: tf.constant(output_keep_prob),
-        false_fn=lambda: tf.constant(1.0))
-    state_keep_prob = tf.cond(
-        pred=is_training,
-        true_fn=lambda: tf.constant(state_keep_prob),
-        false_fn=lambda: tf.constant(1.0))
-    return tf.compat.v1.nn.rnn_cell.DropoutWrapper(
-        cell,
-        output_keep_prob=output_keep_prob,
-        state_keep_prob=state_keep_prob)
-
-
 def bidi_rnn_layers(
         is_training,
         inputs,
         num_layers=1,
         output_size=50,
         output_keep_prob=1.0,
-        state_keep_prob=1.0,
-        seq_lens=None,
         gru=False,
-        residual_connections=False):
+        residual_connections=False,
+        return_sequences=True,
```

Comment:
Is this option needed? We are not offering sentence level classification (yet?), right?

Reply:
It's needed for constructing the char (byte) RNN representations.

Reply:
Ah, thanks for clarifying.